pages/common/git-mailinfo.md

```diff
@@ -0,0 +1,16 @@
+# git mailinfo
+
+> Extract patch and authorship information from a single email message.
+> More information: <https://git-scm.com/docs/git-mailinfo>.
+
+- Extract the patch and author data from an email message:
+
+`git mailinfo {{message|patch}}`
```

Comment:
Is this a path to a file, or something else? Same for the other examples.

Reply:
`mailinfo` is very obscure. It's one of those `git` functions that's not very well documented and I haven't been able to test it out properly yet.